Patches/Vanilla Weapons Expanded - Laser/RangedSpacer.xml

```diff
@@ -208,8 +208,8 @@
           <Mass>2.25</Mass>
           <Bulk>6.90</Bulk>
           <SwayFactor>0.92</SwayFactor>
-          <ShotSpread>0.06</ShotSpread>
-          <SightsEfficiency>0.8</SightsEfficiency>
+          <ShotSpread>0.04</ShotSpread>
+          <SightsEfficiency>1.10</SightsEfficiency>
```

Comment:
The pistol is one-handed and so should keep the one-handed sights value

Reply:
'Tis not a pistol. :)